### PR TITLE
Simplify ERC7575

### PR DESCRIPTION
**Pull request by hieronx**
When opening a pull request to submit a new EIP, please use the suggested template: https://github.com/ethereum/EIPs/blob/master/eip-template.md

We have a GitHub bot that automatically merges some PRs. It will merge yours immediately if certain criteria are met:

 - The PR edits only existing draft PRs.
 - The build passes.
 - Your GitHub username or email address is listed in the 'author' header of all affected PRs, inside <triangular brackets>.
 - If matching on email address, the email address is the one publicly listed on your GitHub profile.

### CODE REVIEW

File: ERCS/erc-7575.md [hunk 1]
```diff
@@ -74,74 +58,28 @@ MUST _NOT_ revert.
...
 
 ### Ability to externalize [ERC-20](./eip-20.md) Dependency
-By allowing `share != address(this)`, the Vault can have an external contract managing the [ERC-20](./eip-20.md) functionality of the Share. In the case of Multi-Vaults, this avoids the confusion that might arise if each Vault itself were required to be an [ERC-20](./eip-20.md), which could cause confusion for integrators and front-ends. This approach also enables the creation of new types of Vaults, such as Pipes, which facilitate the conversion between two external [ERC-20](./eip-20.md) tokens.
+By allowing `share != address(this)`, the Vault can have an external contract managing the [ERC-20](./eip-20.md) functionality of the Share. In the case of Multi-Asset, this avoids the confusion that might arise if each Vault itself were required to be an [ERC-20](./eip-20.md), which could cause confusion for integrators and front-ends. This approach also enables the creation of new types of Vaults, such as Pipes, which facilitate the conversion between two external [ERC-20](./eip-20.md) tokens.
```
Comment (vikramarun) [inline ERCS/erc-7575.md:117]: Thoughts on including more about Pipes here, to give the reader some context on how we originally saw them being used? Nothing about partial vaults explicitly but something like

These Pipes could be unidirectional (i.e. only for assets to shares via deposit/mint, or shares to assets via redeem/withdraw) or bidirectional for both entry and exit flows.

**Review by vikramarun — APPROVED** [review]

File: ERCS/erc-7575.md (hunk 1 above)
Comment (hieronx) [reply ERCS/erc-7575.md:117]: Re-added the pipes content!

**Review by vikramarun — APPROVED** [review]
lgtm!

**Review by 0xfarhaan — COMMENTED** [review]
LGTM cheers for pushing this along

**Pull request merged**